nested_eagle/ursa/scientific_workflow/verification/grid2obs/submit_obs_verification.sh

```diff
@@ -19,5 +19,5 @@ export WORKDIR_BASE_PATH=$PWD
 sed -i "/^.*workdir:.*$/c\  workdir: $WORKDIR_BASE_PATH\/wxvx_workdir\/lam" wxvx_lam.yaml
 sed -i "/^.*workdir:.*$/c\  workdir: $WORKDIR_BASE_PATH\/wxvx_workdir\/global" wxvx_global.yaml
 
-wxvx -c wxvx_lam.yaml -t plots
-wxvx -c wxvx_global.yaml -t plots
+srun wxvx -c wxvx_lam.yaml -t plots -n 4
```

Comment:
```suggestion
srun wxvx -c wxvx_global.yaml -t plots
```
srun will inherit number tasks from sbatch